static/app/gettingStartedDocs/android/android.tsx

```diff
@@ -93,6 +95,24 @@ val breakWorld = Button(this).apply {
 addContentView(breakWorld, ViewGroup.LayoutParams(
   ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT))`;
 
+const getReplaySetupSnippetKotlin = (params: Params) => `
+SentryAndroid.init(context) { options ->
+  options.dsn = "${params.dsn.public}"
+  options.isDebug = true
+
+  // Currently under experimental options:
+  options.experimental.sessionReplay.errorSampleRate = 1.0
```

Comment:
FYI: i've already merged the PR that changes the name to `onErrorSampleRate` - do we wanna wait for the new SDK release and change it now, or rather ship this and follow-up later?

Reply:
hmm @romtsn let's follow up perhaps, since we need to update the docs anyway - can do both at the same time. feel free to ping me to update when the SDK release is out